dodo.py

```diff
@@ -0,0 +1,21 @@
+"""doit Script.
+
+```py
+# Ensure that packages are installed
+poetry install
+# List Tasks
+poetry run doit list
+# (Or use a poetry shell)
+# > poetry shell
+# (shell) doit list
+
+# Run tasks individually (examples below)
+(shell) doit run test
+(shell) doit run test check
+# Or all of the tasks in DOIT_CONFIG
+(shell) doit
+```
+
+"""
+
+from pattern_feedback_tool.doit_tasks import * # noqa: F401
```

Comment:
I couldn't get it to work. When I run `doit play` I get the message `command not found: doit`

I'm not sure if `pattern_feedback_tool` has `doit_tasks`

Reply:
It needs the code from this PR: https://github.com/DesignPatternsAdventure/pattern_feedback_tool/pull/2

I pushed changes that should work now once you run `poetry install --sync`